zebra-state/src/service.rs

```diff
@@ -154,46 +157,38 @@ impl StateService {
         self.mem.any_chain_contains(hash) || &self.disk.finalized_tip_hash() == hash
     }
 
-    /// Returns true if the given hash has been committed to either the finalized
-    /// or non-finalized state.
-    fn contains_committed_block(&self, block: &Block) -> bool {
-        let hash = block.hash();
-        let height = block
-            .coinbase_height()
-            .expect("coinbase heights should be valid");
-
-        self.mem.any_chain_contains(&hash) || self.disk.hash(height) == Some(hash)
-    }
-
     /// Attempt to validate and commit all queued blocks whose parents have
     /// recently arrived starting from `new_parent`, in breadth-first ordering.
-    #[instrument(skip(self))]
     fn process_queued(&mut self, new_parent: block::Hash) {
         let mut new_parents = vec![new_parent];
 
-        while let Some(parent) = new_parents.pop() {
-            let queued_children = self.queued_blocks.dequeue_children(parent);
+        while let Some(parent_hash) = new_parents.pop() {
+            let queued_children = self.queued_blocks.dequeue_children(parent_hash);
 
-            for QueuedBlock { block, rsp_tx } in queued_children {
-                let hash = block.hash();
+            for (child, rsp_tx) in queued_children {
+                let child_hash = child.hash.clone();
```

Comment:
Stable clippy says...
```suggestion
                let child_hash = child.hash;
```